uaclient/tests/test_util.py

```diff
@@ -575,9 +573,10 @@ def test_fallback_used_if_all_other_files_are_empty(
         self, FakeConfig, tmpdir, empty_value
     ):
         data_machine_id = tmpdir.join("machine-id")
-        cfg = FakeConfig.for_attached_machine()
-        # Cache it in cfg, so the load_file mock does not mess everything up
-        cfg.machine_token
+        cfg = FakeConfig()
```

Comment:
Can't we just do something like this here:
```python
FakeConfig().for_attached_machine(
    machine_token={"test": "blah"}
)
```

Reply:
:man_facepalming:

Reply:
had to initialize the property as well because of the mock, but it's clearer for sure